test/fixtures/label/position.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  tolerance: 0.0060,
+  config: {
+    type: 'bar',
+    options: {
+      scales: {
+        x: {
+          labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+        },
+        y: {
+          display: false,
+          min: 0,
+          max: 25
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            text1: {
+              type: 'label',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xValue: 'January',
+              yValue: 20,
+              backgroundColor: 'transparent',
+              borderWidth: 0,
+              content: 'This is my text'
```

Comment:
Could use the position value as label, for example: `'poistion: undefined'`
Could also place all on same value and maybe make this a line chart with a line on that value, so there would be a reference for the eye.

I'm not 100% `position` is the correct property name for this functionality?